feat: learn english block 6.2 (#52539)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-your-progress-in-weekly-stand-up-meetings/6579e0385253cd3a7bd44902.md

---
id: 6579e0385253cd3a7bd44902
title: Task 49
challengeType: 22
dashedName: task-49
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
---

<!--
AUDIO REFERENCE: 
Sophie: "I would like to _ that we create coding _ to ensure _ across our projects."
-->

# --description--

> 'Guidelines' are instructions or rules that indicate how something should be done. For instance, 'coding guidelines' help programmers understand how to write code for a project. 'Consistency' means keeping things the same or uniform, especially in quality. In a work context, 'ensuring consistency' might mean making sure that all parts of a project follow the same standards.

Reviewer: ```suggestion
`Guidelines` are instructions or rules that indicate how something should be done. For instance, `coding guidelines` help programmers understand how to write code for a project. `Consistency` means keeping things the same or uniform, especially in quality. In a work context, `ensuring consistency` might mean making sure that all parts of a project follow the same standards.
```

# --fillInTheBlank--

## --sentence--

> `Sophie: "I would like to _ that we create coding _ to ensure _ across our projects."`

Reviewer: ```suggestion
`I would like to _ that we create coding _ to ensure _ across our projects.`
```

## --blanks--

`suggest`

### --feedback--

> To 'suggest' means to propose an idea or plan, here referring to Sophie's idea about creating guidelines.

Reviewer: ```suggestion
To `suggest` means to propose an idea or plan, here referring to Sophie's idea about creating guidelines.
```

---

`guidelines`

### --feedback--

'Guidelines' refer to rules or instructions, in this case, for coding in projects.

---

`consistency`

### --feedback--

> 'Consistency' here refers to maintaining uniform standards or practices across projects.

Reviewer: ```suggestion
`Consistency` here refers to maintaining uniform standards or practices across projects.
```
